woodwork/utils.py

```diff
@@ -264,6 +264,23 @@ def _to_latlong_float(val):
         raise ValueError(f'Latitude and Longitude values must be in decimal degrees. The latitude or longitude represented by {val} cannot be converted to a float.')
 
 
+def _is_valid_latlong_series(series):
+    '''Returns True if all elements in the series contain properly formatted LatLong values,
+    otherwise returns False'''
+    if series.apply(_is_valid_latlong_value).all():
+        return True
+    return False
+
+
+def _is_valid_latlong_value(val):
+    '''Returns True if the value provided is a properly formatted LatLong value, otherwise
+    returns False'''
+    if isinstance(val, tuple) and len(val) == 2:
```

Comment:
Something to make note of for #493, but we'll need the definition of a valid latlong to use lists instead of tuples for koalas

Reply:
Added a note in that issue